Update semgrep-intellij.md (#2279)
semgrep/semgrep-docs: docs/extensions/semgrep-intellij.md

$ python3 -m pip install semgrep
```

> > Semgrep's IntelliJ extension is currently in public BETA on Windows machines.

Reviewer: ```suggestion
> Semgrep's IntelliJ extension for Windows users is currently in **public beta**.
```
